2-mandatory/4-people-i-know.js

```diff
@@ -393,25 +393,33 @@ You can see who people's colleagues are by seeing the "colleagues" array in each
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
-let friendsWhoAreColleaguesOfStacie = [];
+const friendsWhoAreColleaguesOfStacie = friends.filter(({ colleagues }) => colleagues.some(({ name }) => name === "Stacie Villarreal")).map(({ name: { first, last } }) => `${first} ${last}`);
 /*
 5) Find "Multi-tasking" colleagues
 Next, I want you to find all of the colleagues of my friends who are good at "Multi-tasking"
 You can tell if they are good at "Multi-tasking" because they will have it listed in their skills
 This time, I only want the full names of the people who can multitask
 */
 
-let colleaguesWhoCanMultitask = [];
+const colleaguesWhoCanMultitask = [];
 
-/* ======= TESTS - DO NOT MODIFY ===== 
+friends.forEach(({ colleagues }) => {
+  colleagues.forEach(({ name, skills }) => {
+    if (skills.includes("Multi-tasking")) {
+      colleaguesWhoCanMultitask.push(name);
+    }
+  });
+});
```

Comment:
Nice approach overall,  can you also think of a way to use a `map `and` reduce` approach, rather than `forEach`?

Reply:
I have created added a 2nd method with reduce, filter and map.
is that what you meant?

Reply:
Nice, looks good 👍